app/src/main/res/layout/fragment_example.xml

```diff
@@ -1,45 +1,118 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/linearLayout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:padding="@dimen/spacing_large"
-    android:gravity="center"
-    android:orientation="vertical">
+    android:gravity="center">
 
     <TextView
-        android:layout_width="match_parent"
-        android:layout_height="@dimen/spacing_none"
-        android:layout_weight="1"
-        android:layout_margin="@dimen/spacing_medium_more"
-        android:gravity="top|center_horizontal"
-        android:text="@string/example_title"
-        android:textSize="@dimen/font_medium_extra" />
+        android:id="@+id/textView"
+        android:layout_width="0dp"
+        android:layout_height="230dp"
```

Comment:
We should add the hardcode dimens in the dimens.xml file.

Reply:
solved in login-backend-access (https://github.com/wolox-training/ph-android/pull/4)